### PR TITLE
Set pip button

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Based on latest article changes, I've updated this sample.

You can try a live preview at https://beaufortfrancois.github.io/samples/picture-in-picture/
I'd recommend setting `Slow 3G` in DevTools Network panel to observe new changes.
@mounirlamouri @japacible

### CODE REVIEW

File: picture-in-picture/index.js [hunk 1]
```diff
@@ -31,3 +29,18 @@ function onPipWindowResize(event) {
...
+
+// Set button ability if Picture-in-Picture can be used.
+setPipButton();
+
+video.addEventListener('loadedmetadata', setPipButton);
+video.addEventListener('emptied', setPipButton);
+
+function setPipButton() {
+  togglePipButton.disabled = true ? (video.readyState === 0) :
+      !document.pictureInPictureEnabled || video.disablePictureInPicture;
```
Comment (mounirlamouri) [inline picture-in-picture/index.js]: `true ?` means you will only check for the `readyState` or am I missing something?

File: picture-in-picture/index.js [hunk 2]
```diff
@@ -31,3 +29,18 @@ function onPipWindowResize(event) {
   const pipWindow = event.target;
   log(`> Window size is ${pipWindow.width}x${pipWindow.height}`);
 }
+
+
+// Hide button if Picture-in-Picture is not supported.
+togglePipButton.hidden = !('pictureInPictureEnabled' in document);
```
Comment (mounirlamouri) [inline picture-in-picture/index.js]: Should this be: 
```javascript
if ('pictureInPictureEnabled' in document) {
  setPipButton();
  // add listeners and all ...
} else {
  togglePipButton. hidden = true;
}
```

Comment (beaufortfrancois) [reply picture-in-picture/index.js]: Done.

File: picture-in-picture/index.js (hunk 1 above)
Comment (beaufortfrancois) [reply picture-in-picture/index.js]: I was thinking about checking `readyState === 0` only but this ternary thing makes it harder to read. So I've removed it.

Thanks!

**Review by mounirlamouri — APPROVED** [review]

**Pull request merged**